runtimeconfig/manager.go

```diff
@@ -35,7 +37,7 @@ type Config struct {
 
 // RegisterFlags registers flags.
 func (mc *Config) RegisterFlags(f *flag.FlagSet) {
-	f.StringVar(&mc.LoadPath, "runtime-config.file", "", "File with the configuration that can be updated in runtime.")
+	f.StringVar(&mc.LoadPath, "runtime-config.file", "", "Comma separated list of yaml files with the configuration that can be updated at runtime.")
```

Comment:
Considered changing this to 'files' from 'file' but that would have a rippling effect to all configs today. If there is a strong opinion it can be changed.